Misc/NEWS.d/next/Library/2023-10-25-05-01-28.gh-issue-111293.FSsLT6.rst

```diff
@@ -0,0 +1 @@
+Fix os.DirEntry.inode() dropping higher 64 bits of a file id on Windows.
```

Comment:
```suggestion
Fix :data:`os.DirEntry.inode` dropping higher 64 bits of a file id on some filesystems on Windows.
```

Hopefully that link resolves, I'm not 100% sure it's the right one.

Reply:
Also move this entry into the "Windows" directory instead of "Library" (just moving the file is fine)